setup.py

```diff
@@ -1,33 +1,34 @@
-import sys
 import os
+import sys
```

Comment:
The changes in this file look like they're coming from `black` / `isort` / ... Is that the case? Just double checking the changes here are formatting-only

Reply:
yes -- formatting only. I can revert those changes in try to keep them in #29 if you think that's more appropriate